packages/plugins/graphql/server/services/internals/types/response-collection-meta.js

```diff
@@ -25,8 +25,9 @@ module.exports = ({ strapi }) => {
             const { args, resourceUID } = parent;
             const { start, limit } = args;
             const safeLimit = Math.max(limit, 1);
+            const contentType = strapi.contentTypes[resourceUID];
```

Comment:
We should use .getModel instead of .contentTypes, I'll make the change as @innerdvations is off

Reply:
Done :heavy_check_mark: